pydantic_ai_slim/pydantic_ai/messages.py

```diff
@@ -1649,9 +1675,9 @@ class FinalResultEvent:
 
 
 ModelResponseStreamEvent = Annotated[
-    PartStartEvent | PartDeltaEvent | FinalResultEvent, pydantic.Discriminator('event_kind')
+    PartStartEvent | PartDeltaEvent | PartEndEvent | FinalResultEvent, pydantic.Discriminator('event_kind')
```

Comment:
Probably worth a note here that we may add additional cases to this in the future and you should not assume this is comprehensive. (Given we are doing that now.)